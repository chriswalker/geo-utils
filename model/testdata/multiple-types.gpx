<?xml version="1.0" encoding="UTF-8"?>
<gpx version="1.1" creator="Chris Walker" xmlns="http://www.topografix.com/GPX/1/1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd">
	<trk>
		<name>Covent Garden Loop Track</name>
		<desc>Length: 3.795 km (2.358 mi)</desc>
		<trkseg>
			<trkpt lat="51.5079679" lon="-0.1265359"></trkpt>
			<trkpt lat="51.5096239" lon="-0.1232529"></trkpt>
			<trkpt lat="51.5111463" lon="-0.1192832"></trkpt>
			<trkpt lat="51.5112131" lon="-0.118897"></trkpt>
			<trkpt lat="51.5116805" lon="-0.1188111"></trkpt>
			<trkpt lat="51.5124951" lon="-0.1185536"></trkpt>
			<trkpt lat="51.5129892" lon="-0.1177812"></trkpt>
			<trkpt lat="51.5131895" lon="-0.1170516"></trkpt>
			<trkpt lat="51.5132563" lon="-0.116086"></trkpt>
			<trkpt lat="51.5130693" lon="-0.1153779"></trkpt>
			<trkpt lat="51.5129491" lon="-0.1149058"></trkpt>
			<trkpt lat="51.5127221" lon="-0.1145196"></trkpt>
			<trkpt lat="51.5128957" lon="-0.1135325"></trkpt>
			<trkpt lat="51.5133765" lon="-0.1127386"></trkpt>
			<trkpt lat="51.5136702" lon="-0.110786"></trkpt>
			<trkpt lat="51.5139507" lon="-0.1109147"></trkpt>
			<trkpt lat="51.5153928" lon="-0.1114726"></trkpt>
			<trkpt lat="51.5161673" lon="-0.1118803"></trkpt>
			<trkpt lat="51.5172088" lon="-0.1129103"></trkpt>
			<trkpt lat="51.5179699" lon="-0.113554"></trkpt>
			<trkpt lat="51.5181836" lon="-0.1136613"></trkpt>
			<trkpt lat="51.5181168" lon="-0.1146269"></trkpt>
			<trkpt lat="51.5178498" lon="-0.1169658"></trkpt>
			<trkpt lat="51.517783" lon="-0.1180816"></trkpt>
			<trkpt lat="51.5176495" lon="-0.1186395"></trkpt>
			<trkpt lat="51.5176228" lon="-0.1189828"></trkpt>
			<trkpt lat="51.5176762" lon="-0.1192403"></trkpt>
			<trkpt lat="51.5175694" lon="-0.120163"></trkpt>
			<trkpt lat="51.5175026" lon="-0.1205921"></trkpt>
			<trkpt lat="51.5173157" lon="-0.1217508"></trkpt>
			<trkpt lat="51.5165946" lon="-0.1212144"></trkpt>
			<trkpt lat="51.5159537" lon="-0.1205063"></trkpt>
			<trkpt lat="51.5149922" lon="-0.1218796"></trkpt>
			<trkpt lat="51.5145249" lon="-0.1221585"></trkpt>
			<trkpt lat="51.5139106" lon="-0.1231241"></trkpt>
			<trkpt lat="51.5137771" lon="-0.1235104"></trkpt>
			<trkpt lat="51.5136235" lon="-0.123682"></trkpt>
			<trkpt lat="51.5132095" lon="-0.1243901"></trkpt>
			<trkpt lat="51.5126286" lon="-0.1253772"></trkpt>
			<trkpt lat="51.5121279" lon="-0.1263964"></trkpt>
			<trkpt lat="51.511981" lon="-0.126965"></trkpt>
			<trkpt lat="51.5118274" lon="-0.126965"></trkpt>
			<trkpt lat="51.5117272" lon="-0.1270294"></trkpt>
			<trkpt lat="51.5108125" lon="-0.1269007"></trkpt>
			<trkpt lat="51.5095838" lon="-0.126847"></trkpt>
			<trkpt lat="51.5094636" lon="-0.126847"></trkpt>
			<trkpt lat="51.5093969" lon="-0.127008"></trkpt>
			<trkpt lat="51.5094102" lon="-0.1271689"></trkpt>
			<trkpt lat="51.5087692" lon="-0.1271045"></trkpt>
			<trkpt lat="51.5079679" lon="-0.1269865"></trkpt>
		</trkseg>
	</trk>
	<rte>
		<name>Parliament Loop Route</name>
		<desc>Length: 1.722 km (1.07 mi)</desc>
			<rtept lat="51.5072734" lon="-0.127126"></rtept>
			<rtept lat="51.5071866" lon="-0.1265359"></rtept>
			<rtept lat="51.5069395" lon="-0.1252377"></rtept>
			<rtept lat="51.5066924" lon="-0.123843"></rtept>
			<rtept lat="51.5064386" lon="-0.1226306"></rtept>
			<rtept lat="51.5041881" lon="-0.1234674"></rtept>
			<rtept lat="51.5026254" lon="-0.1239181"></rtept>
			<rtept lat="51.5010225" lon="-0.1243365"></rtept>
			<rtept lat="51.5010559" lon="-0.125978"></rtept>
			<rtept lat="51.5027055" lon="-0.1259351"></rtept>
			<rtept lat="51.5040546" lon="-0.1260853"></rtept>
			<rtept lat="51.5050429" lon="-0.1264071"></rtept>
			<rtept lat="51.5057909" lon="-0.1267719"></rtept>
			<rtept lat="51.5065121" lon="-0.1271582"></rtept>
			<rtept lat="51.5069662" lon="-0.1273298"></rtept>
	</rte>
	<wpt lat="51.5130827" lon="-0.1242721">
		<name>Covent Garden Underground Station</name>
		<sym>Circle, Blue</sym>
	</wpt>
	<wpt lat="51.5114668" lon="-0.1282954">
		<name>Leicester Square Underground Station</name>
		<sym>Circle, Blue</sym>
	</wpt>
</gpx>
