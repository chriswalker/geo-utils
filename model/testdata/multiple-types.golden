{"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"MultiPoint","coordinates":[[-0.1242721,51.5130827,0],[-0.1282954,51.5114668,0]]}},{"type":"Feature","geometry":{"type":"MultiLineString","coordinates":[[[-0.1265359,51.5079679,0],[-0.1232529,51.5096239,0],[-0.1192832,51.5111463,0],[-0.118897,51.5112131,0],[-0.1188111,51.5116805,0],[-0.1185536,51.5124951,0],[-0.1177812,51.5129892,0],[-0.1170516,51.5131895,0],[-0.116086,51.5132563,0],[-0.1153779,51.5130693,0],[-0.1149058,51.5129491,0],[-0.1145196,51.5127221,0],[-0.1135325,51.5128957,0],[-0.1127386,51.5133765,0],[-0.110786,51.5136702,0],[-0.1109147,51.5139507,0],[-0.1114726,51.5153928,0],[-0.1118803,51.5161673,0],[-0.1129103,51.5172088,0],[-0.113554,51.5179699,0],[-0.1136613,51.5181836,0],[-0.1146269,51.5181168,0],[-0.1169658,51.5178498,0],[-0.1180816,51.517783,0],[-0.1186395,51.5176495,0],[-0.1189828,51.5176228,0],[-0.1192403,51.5176762,0],[-0.120163,51.5175694,0],[-0.1205921,51.5175026,0],[-0.1217508,51.5173157,0],[-0.1212144,51.5165946,0],[-0.1205063,51.5159537,0],[-0.1218796,51.5149922,0],[-0.1221585,51.5145249,0],[-0.1231241,51.5139106,0],[-0.1235104,51.5137771,0],[-0.123682,51.5136235,0],[-0.1243901,51.5132095,0],[-0.1253772,51.5126286,0],[-0.1263964,51.5121279,0],[-0.126965,51.511981,0],[-0.126965,51.5118274,0],[-0.1270294,51.5117272,0],[-0.1269007,51.5108125,0],[-0.126847,51.5095838,0],[-0.126847,51.5094636,0],[-0.127008,51.5093969,0],[-0.1271689,51.5094102,0],[-0.1271045,51.5087692,0],[-0.1269865,51.5079679,0]]]}},{"type":"Feature","geometry":{"type":"MultiLineString","coordinates":[[[-0.127126,51.5072734,0],[-0.1265359,51.5071866,0],[-0.1252377,51.5069395,0],[-0.123843,51.5066924,0],[-0.1226306,51.5064386,0],[-0.1234674,51.5041881,0],[-0.1239181,51.5026254,0],[-0.1243365,51.5010225,0],[-0.125978,51.5010559,0],[-0.1259351,51.5027055,0],[-0.1260853,51.5040546,0],[-0.1264071,51.5050429,0],[-0.1267719,51.5057909,0],[-0.1271582,51.5065121,0],[-0.1273298,51.5069662,0]]]}}]}